pkg/etl/materialized_view_refresher.go

```diff
@@ -0,0 +1,122 @@
+package etl
```

Comment:
In my experience code like this causes problems. Matview refreshes are slow... / less predictable. it's probably fine, but i don't love it, so wanted to note